qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -899,7 +899,18 @@ def _inject_userscripts(self):
             # @run-at (and @include/@exclude/@match) is parsed by
             # QWebEngineScript.
             new_script = QWebEngineScript()
-            new_script.setWorldId(QWebEngineScript.MainWorld)
+            try:
+                world = int(script.jsworld)
+            except ValueError:
+                try:
+                    world = [v for k, v in _JS_WORLD_MAP.items()
+                             if k.name == script.jsworld.lower()][0]
+                except IndexError:
+                    log.greasemonkey.error(
+                        "script {} has invalid value for '@qute-js-world'"
+                        ": {}".format(script.name, script.jsworld))
+                    continue
```

Comment:
I chose to log an ERROR and refuse to add the script, rather than falling back to the default, because we don't have to be flexible with this input. If someone has provided an unmappable name then we know it isn't a valid value that we just happen to not support because qute is the only target it is valid for. So yell at them to fix it.